pkgs/applications/editors/eclipse/default.nix

```diff
@@ -271,4 +271,19 @@ in {
     };
   };
 
+  scala_ide = buildEclipse {
+    name = "scala-ide";
+    description = "Scala IDE for Eclipse";
+    sources = {
+      "x86_64-linux" = fetchurl {
+          url = http://downloads.typesafe.com/scalaide-pack/4.0.0.vfinal-luna-211-20150305/scala-SDK-4.0.0-vfinal-2.11-linux.gtk.x86_64.tar.gz;
+          sha256 = "b65c5e8160e72c8389537e9e427138e6daa2065f9df3a943a86e40dd1543dd83";
```

Comment:
Why are these hashes longer than the other ones in this file?  I created them with `sha256sum` -- is there another way?